code/modules/projectiles/pins.dm

```diff
@@ -24,7 +24,7 @@ Firing pins as a rule can't be removed without replacing them, blame a really sh
 
 
 
-/obj/item/device/firing_pin/New(newloc)
+/obj/item/device/firing_pin/Initialize(newloc)
 	..()
```

Comment:
`. = ..()`, and `newloc` should be `mapload`. Any ref to `newloc` in this proc should be replaced with `loc`.